### PR TITLE
[v8.0] feat (TS): add getTransformationFilesAsJsonString

### PR DESCRIPTION
**Pull request by chaen**
BEGINRELEASENOTES

*Core
FIX: Returning DISET responses with >100000000 bytes

*Transformation
NEW: Use getTransformationFilesAsJsonString for faster getTransformationFiles

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/TransformationSystem/Service/TransformationManagerHandler.py [hunk 1]
```diff
@@ -302,8 +305,22 @@ def export_getTransformationFiles(
...
             columns=columns,
+            include_web_records=include_web_records,
         )
 
+    types_getTransformationFilesAsJsonString = types_getTransformationFiles
+
+    def export_getTransformationFilesAsJsonString(self, *args, **kwargs):
+        """
+        DEncode cannot cope with nested structures of multiple millions items.
+        Encode everything as a json string, that DEncode cna then transmit faster.
```
Comment (andresailer) [inline src/DIRAC/TransformationSystem/Service/TransformationManagerHandler.py]:
```suggestion
        Encode everything as a json string, that DEncode can then transmit faster.
```

File: src/DIRAC/TransformationSystem/DB/TransformationDB.py [hunk 2]
```diff
@@ -603,6 +603,7 @@ def getTransformationFiles(
         offset=None,
         connection=False,
         columns=None,
+        include_web_records=True,
```
Comment (fstagni) [inline src/DIRAC/TransformationSystem/DB/TransformationDB.py:606]: I would reverse this flag, and make a tiny WebApp patch.

File: src/DIRAC/TransformationSystem/Service/TransformationManagerHandler.py [hunk 3]
```diff
@@ -302,8 +305,22 @@ def export_getTransformationFiles(
...
+
+        This will be the default as of v9.0
```
Comment (fstagni) [inline src/DIRAC/TransformationSystem/Service/TransformationManagerHandler.py:318]: Then you should add a deprecation notice for `export_getTransformationFiles`

File: src/DIRAC/TransformationSystem/DB/TransformationDB.py (hunk 2 above)
Comment (chrisburr) [reply src/DIRAC/TransformationSystem/DB/TransformationDB.py:606]: We suspect it isn't used at all. The idea would be to switch it back in v9 but it's a nasty backwards incompatible change in a patch release if anyone does happen to be using it.

File: src/DIRAC/TransformationSystem/Service/TransformationManagerHandler.py (hunk 3 above)
Comment (chrisburr) [reply src/DIRAC/TransformationSystem/Service/TransformationManagerHandler.py:318]: We were thinking we would keep `getTransformationFiles` and deprecate `getTransformationFilesAsJson` in v9.

**Review by fstagni — APPROVED** [review]

**Pull request merged**